packages/web/src/common/values/token-constant.ts

```diff
@@ -44,6 +46,7 @@ export const GNOT_TOKEN_DEFAULT: TokenModel = {
   logoURI:
     "https://raw.githubusercontent.com/onbloc/gno-token-resource/main/gno-native/images/gnot.svg",
   priceID: "gnot",
```

Comment:
Please remove this item to avoid confusion.

Reply:
i cannot remove until api update, this field exist in current model